### PR TITLE
Pcieutil to load the platform api first instead of using common api

### PR DESCRIPTION
**Pull request by sujinmkang**
#### What I did
Pcieutil to load the platform api first instead of using common api
Some platform device with different BIOS version needs more than one pcie configuration to check the pcie devices properly.
Please refer to the platform api support : https://github.com/Azure/sonic-platform-common/pull/195

#### How I did it
Load the platform pcie api first prior to use the common api

### CODE REVIEW

**Comment by jleveque** [comment]
Can you please update the description to state why this change is necessary?

File: pcieutil/main.py [hunk 1]
```diff
@@ -49,13 +49,18 @@ def load_platform_pcieutil():
...
     try:
-        platform_path, _ = device_info.get_paths_to_platform_and_hwsku_dirs()
-        from sonic_platform_base.sonic_pcie.pcie_common import PcieUtil
-        platform_pcieutil = PcieUtil(platform_path)
+        from sonic_platform.pcie import Pcie
+        platform_pcieutil = Pcie(platform_path)
     except ImportError as e:
-        log.log_error("Failed to load default PcieUtil module. Error : {}".format(str(e)), True)
-        raise e
+        self.log_error("Failed to load platform Pcie module. Error : {}".format(str(e)), True)
```
Comment (ArunSaravananBalachandran) [inline pcieutil/main.py:57]: Please change `self` to `log`
```suggestion
        log.log_error("Failed to load platform Pcie module. Error : {}".format(str(e)), True)
```

**Comment by sujinmkang** [comment]
@jleveque @ArunSaravananBalachandran do you have any other comments?

**Review by jleveque — APPROVED** [review]

**Pull request merged**